test/linalg3.jl

```diff
@@ -161,7 +161,10 @@ for elty in (Float32, Float64, Complex64, Complex128)
                                               0  -0.000000000000002   3.000000000000000])
 end
 
+OLD_STDERR = STDERR
+redirect_stderr()
 for elty in (Float64, Complex{Float64})
+    
```

Comment:
@tkelman Thank you, I submitted a new version. It still fails on AppVeyor, but the problem does not seem related to the code I modified (there's no error at all, actually).